docs/conf.py

```diff
@@ -193,18 +193,18 @@
 #latex_paper_size = 'letter'
 
 # The font size ('10pt', '11pt' or '12pt').
-#latex_font_size = '10pt'
+latex_font_size = '12pt'
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
-  ('index', 'Borg.tex', 'Borg Documentation',
-   'see "AUTHORS" file', 'manual'),
+  ('book', 'Borg.tex', 'Borg Documentation',
+   'The Borg Collective', 'manual'),
```

Comment:
The borg collective is in the table of contents on the next page, I am
not sure that is necessary.

The first page looks really ackward with the current attribution: https://media.readthedocs.org/pdf/borgbackup/latest/borgbackup.pdf It just says "see AUTHORS file" without any context. "The borg collective" looks like a person or a group, so it's a little clearer it's an authorship or attribution.

I'm fine with adding "see authors section" or "see AUTHORS file", but keep in mind that the "authors" section is actually called "The Borg Collective" right now, and "the AUTHORS file" doesn't mean anything for someone browsing the HTML, PDF or ePUB versions of the documentaiton.

So I am not sure it would make sense to keep a reference there at all.